packages/create-toolpad-app/src/index.ts

```diff
@@ -71,23 +71,23 @@ const validatePath = async (relativePath: string): Promise<boolean | string> =>
   const absolutePath = path.join(process.cwd(), relativePath);
 
   try {
-    await fs.mkdir(absolutePath);
+    await fs.access(absolutePath, fs.constants.F_OK);
 
-    return true;
-  } catch {
     // Directory exists, verify if it's empty to proceed
-    try {
-      if (await isFolderEmpty(absolutePath)) {
-        return true;
-      }
-      return `${chalk.red('error')} - The directory at ${chalk.blue(
-        absolutePath,
-      )} contains files that could conflict. Either use a new directory, or remove conflicting files.`;
-    } catch {
-      return `${chalk.red('error')} - Unable to access directory at ${chalk.red(
-        absolutePath,
-      )}. Please provide a different path.`;
+    if (await isFolderEmpty(absolutePath)) {
+      return true;
     }
+    return `${chalk.red('error')} - The directory at ${chalk.blue(
+      absolutePath,
+    )} contains files that could conflict. Either use a new directory, or remove conflicting files.`;
+  } catch (error: any) {
```

Comment:
```suggestion
  } catch (rawError: unkown) {
    const error = errorFrom(rawError);
```

Reply:
Do you recommend importing `errorFrom` from `@mui/toolpad-core` or rewriting that function here?

Reply:
Added something local to avoid adding dependencies to this package